deny.toml

```diff
@@ -43,6 +43,7 @@ allow = [
     "0BSD",
     "BSD-2-Clause",
     "BSD-3-Clause",
+    "BSL-1.0",
```

Comment:
For next reader - this is [Boost Source License](https://choosealicense.com/licenses/bsl-1.0/) and is pure OSS.